### PR TITLE
Move System Information to Repairs

### PR DESCRIPTION
**Pull request by zsarnett**
## Proposed change

CleanShot 2022-07-26 at 13 19 53
CleanShot 2022-07-26 at 13 19 43

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: hassio/src/system/hassio-supervisor-info.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ import {
 import {
   UNHEALTHY_REASON_URL,
   UNSUPPORTED_REASON_URL,
```
Comment (bramkragten) [inline hassio/src/system/hassio-supervisor-info.ts:28]: We should move this to data

File: src/panels/config/repairs/dialog-system-information.ts [hunk 2]
```diff
@@ -421,102 +370,77 @@ class HaConfigSystemHealth extends SubscribeMixin(LitElement) {
...
           domainParts.push("</details>");
         }
       }
     }
 
     await copyToClipboard(
-      `${github ? "## " : ""}System Health\n${haContent}\n\n${domainParts.join(
-        "\n\n"
-      )}`
+      `${"## "}System Information\n${haContent}\n\n${domainParts.join("\n\n")}`
```
Comment (bramkragten) [inline src/panels/config/repairs/dialog-system-information.ts:392]:
```suggestion
      `## System Information\n${haContent}\n\n${domainParts.join("\n\n")}`
```

File: src/panels/config/repairs/ha-config-repairs-dashboard.ts [hunk 3]
```diff
@@ -23,13 +39,54 @@ class HaConfigRepairsDashboard extends LitElement {
...
+  @state() private _coreStats?: HassioStats;
+
   private _getFilteredIssues = memoizeOne(
     (showIgnored: boolean, repairsIssues: RepairsIssue[]) =>
       showIgnored
         ? repairsIssues
         : repairsIssues.filter((issue) => !issue.ignored)
   );
 
+  public hassSubscribe(): Array<UnsubscribeFunc | Promise<UnsubscribeFunc>> {
```
Comment (bramkragten) [inline src/panels/config/repairs/ha-config-repairs-dashboard.ts:57]: We always stay subscribed, but only send the info in the dialog, without updating the data in the dialog when it updates?

We should move the subscribe to the dialog.

**Review by bramkragten — APPROVED** [review]

**Comment by frenck** [comment]
Would this work with all existing places linking to the system health? For example, the fast amount of My buttons and links from e.g., issue trackers to that page to ask the user to collect data?

**Comment by ludeeus** [comment]
https://www.home-assistant.io/integrations/system_health/

https://www.home-assistant.io/more-info/system-health/

Needs an update

**Pull request merged**